src/app/src/store/store.js

```diff
@@ -1,11 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit';
 
+import api from '../api';
 import authReducer from './authSlice';
 import mapReducer from './mapSlice';
```

Comment:
Should we also be importing `errorSlice` here?

Reply:
15f1ffefe859b48e89152a38e56ce203218b8fb9